tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -498,8 +499,12 @@ public String toJSONString() throws JSONException {
                 .toString();
         }
 
+        public boolean canImportOldVotes(CheckCLDR.Phase inPhase) {
+            return UserRegistry.userIsVetter(this) && (inPhase == Phase.SUBMISSION);
+        }
+
         public boolean canImportOldVotes() {
-            return UserRegistry.userIsVetter(this) && (CLDRConfig.getInstance().getPhase() == Phase.SUBMISSION);
+            return canImportOldVotes(CLDRConfig.getInstance().getPhase());
```

Comment:
very minor quibble: i'd rather see the subroutine below the caller -- do they both need to be public?

Reply:
One is the one that's publicly used, the other is the implementation split out for testing. (Wanted to make it non phase dependent).  I'll swap them though.